openapi/definitions.yaml

```diff
@@ -106,6 +108,29 @@ NewMessage:
       $ref: "#/FiscalCode"
   required:
     - content
+PrescriptionData:
+  type: object
+  description: |-
+    Metadata needed to process medical prescriptions.
+  properties:
+    nre:
+      $ref: "#/PrescriptionNre"
+    iup:
+      $ref: "#/PrescriptionIup"
+    doctor_fiscal_code:
```

Comment:
in English, it's called `md` (Medicinae Doctor), so it should be `md_fiscal_code`, or perhaps if we want to be a little more generic we can use `prescriber_fiscal_code`.

Reply:
I'm ok with this but since these specs are already under review let's double check with @matteodesanti